integration-templates/discourse/nango.yaml

```
@@ -0,0 +1,18 @@
+integrations:
+    discourse:
+        syncs:
+            users:
+                description: |
+                    Fetches a list of active users from Discourse.
```

I don't think we filter
```suggestion
                    Fetches a list of users from Discourse.
```

---

The implemented endpoint is filtering for active users only  `/active`.

---

ah yeah I didn't saw it was in the path of the endpoint my bad. 
Is it still relevant to have `active` in the User model then? or maybe the sync should be renamed to users-active. Not suggesting to change anything but wondering if we want to be explicit

---

I think that would be nice to explicitly name the script as `active-users` as it only fetches for active users only.